sql/database.sql

```diff
@@ -4756,6 +4759,19 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `activity`) VALUES ('Sort_Direction', '0', 'asc', 10, 1, 1);
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `activity`) VALUES ('Sort_Direction', '1', 'desc', 20, 0, 1);
 
+-- ActEncounterCode [FHIR Encounter.class]
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','AMB');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','EMER');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','FLD','field');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','HH','home health');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','IMP','inpatient encounter');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','ACUTE','inpatient acute');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','NONAC','inpatient non-acute');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','OBSENC','observation encounter');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','PRENC','pre-admission');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','SS','short stay');
+INSERT INTO list_options(list_id,option_id,title) VALUES ('_ActEncounterCode','VR','virtual');
```

Comment:
1. Need an item that gives the list a title:
```
INSERT INTO list_options (list_id, option_id, title, seq) VALUES ('lists', '_ActEncounterCode', 'Value Set ActEncounterCode', 1);
```

2. AMB and EMER need titles

3. Add seq to all the items (start with 10 and increment by 10)

3. Give AMB the is_default
```
INSERT INTO list_options (list_id, option_id, title, seq, is_default) VALUES ('_ActEncounterCode', 'AMB', 'ambulatory', 10, 1);
```

Note that the title is what will show in the selector from the encounter UI, so this will likely need to be capitalized. Because of that, I would store the actual string that is needed for FHIR in the notes column:
```
INSERT INTO list_options (list_id, option_id, title, notes, seq, is_default) VALUES ('_ActEncounterCode', 'AMB', 'Outpatient', 'ambulatory', 10, 1);
```
And would use following titles respectively
Outpatient
Emergency Dept
Out in Field
Home Health
Inpatient Encounter
Inpatient Acute
Inpatient Non-Acute
Observation Encounter
Pre-Admission
Short Stay
Virtual Encounter

I would then place this as an input selector in the encounter UI which will default to AMB and not allow it to be set to empty. I the Encounter UI would place it to the right of Encounter Provider selector.